kafka-connect-aws-s3/src/test/scala/io/lenses/streamreactor/connect/aws/s3/sink/config/S3SinkConfigDefBuilderTest.scala

```diff
@@ -229,6 +229,23 @@ class S3SinkConfigDefBuilderTest extends AnyFlatSpec with MockitoSugar with Matc
     }
   }
 
+  "S3SinkConfigDefBuilder" should "support selecting from *" in {
+    val props = Map(
+      "connect.s3.kcql" -> s"insert into $BucketName:$PrefixName select * from `*` STOREAS `JSON`  PROPERTIES('${DataStorageSettings.StoreEnvelopeKey}'=true, '${FlushCount.entryName}'=1)",
+    )
+
+    config.CloudSinkBucketOptions(connectorTaskId, S3SinkConfigDefBuilder(props)) match {
+      case Left(value) => fail(value.toString)
+      case Right(value) =>
+        value.map(_.dataStorage) should be(List(DataStorageSettings(envelope = true,
+                                                                    key      = true,
+                                                                    value    = true,
+                                                                    metadata = true,
+                                                                    headers  = true,
+        )))
+    }
```

Comment:
The pattern match and `fail()` is not really necessary if you use scalatest's `EitherValues` trait.

```suggestion
    config.CloudSinkBucketOptions(connectorTaskId, S3SinkConfigDefBuilder(props)).value.map(_.dataStorage) should be(
      List(
        DataStorageSettings(
          envelope = true,
          key      = true,
          value    = true,
          metadata = true,
          headers  = true,
        )
      )
    )
```

Reply:
Thanks for changing all those tests!